backend/app/tasks/robot_queue_tasks.py

```
@@ -221,29 +298,79 @@ def _queued_devices(db: Session) -> list[tuple[str, str]]:
-    row = (
-        db.query(Kind)
-        .filter(
-            Kind.kind == "Device",
-            Kind.name == device_id,
-            Kind.is_active == True,
-        )
-        .first()
+    query = db.query(Kind).filter(
+        Kind.kind == "Device",
+        Kind.name == device_id,
+        Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)